src/pkg/leaderElection.go

```diff
@@ -43,8 +44,26 @@ func RunLeaderElection(client *clientset.Clientset, lockName, lockIdentity, lock
 				log.Info().Msgf("Perform Migration")
 				for {
 					log.Info().Msgf("leader is %s", lockIdentity)
-					log.Info().Msgf("Getting replica count...")
-					time.Sleep(5 * time.Second)
+					replicaCount, err := getReplicas()
+					if err != nil {
+						log.Fatal().Msgf("Failed to get replica count")
```

Comment:
we should add a "continue" here so that if an error did happen in `getReplicas` we would proceed onto the next lines of code.

Reply:
Would we want to continue if we didn't get the replica counts?  What would it try to set it to in that case?

Reply:
"continue" here means stop the execution of the for loop and go back to the top.  Its how you stop it from proceeding to the next line.

Reply:
Oh nvm, I thought this was outside of for loop the first time I read it, you're saying continue out of this iteration in the for loop.  👍